service/version_bundles.go

```diff
@@ -0,0 +1,47 @@
+package service
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func newVersionBundles() []versionbundle.Bundle {
+	return []versionbundle.Bundle{
+		{
+			Changelogs: []versionbundle.Changelog{},
+			Components: []versionbundle.Component{
+				{
+					Name:    "calico",
+					Version: "3.0.1",
+				},
+				{
+					Name:    "docker",
+					Version: "1.12.6",
```

Comment:
Docker version is just a copy/paste. Is there an easy way to check this?

Reply:
It comes from the CoreOS version. See https://coreos.com/releases, I am just not sure which CoreOS version is being used right now.

Reply:
It would be maybe nice to add the info about the os as well?

Reply:
We could do that. It is only that we show the users what we define here and we would maybe need to filter certain things. Don't know. Anyway, when we add it in Azure, we should do so for AWS and KVM as well.

Reply:
So here in the version bundle we can't be really sure what the OS (and in consequence etcd) version is. In case of azure it depends on settings in the custom object. This is the same case with AWS AMI. Not sure how it's done for KVM.

My opinion is it should be baked into the bundle. I.e. not configurable with custom object or whatsoever.

cc @r7vme as this may be interesting for you.

Reply:
> My opinion is it should be baked into the bundle. I.e. not configurable with custom object or whatsoever.

+1

Reply:
+1 from me too. We did that with the kubernetes version and k8scloudconfig and I think was the right call.

Reply:
So this version is correct now as the CoreOS version used is 1465.7.0 (kind of stale - from September last year). But the operator doesn't have any control over that.

New coreos versions have  17.09.0 version of docker. Versioning approach changed?

I'm leaving it how it is. This is still wip.

Reply:
yes, docker switched to new versioning scheme, so 17.09 is fine.

regarding this change, CoreOS version is taken from [CRO](https://github.com/giantswarm/azure-operator/blob/master/examples/azure-resource-lab-chart/templates/cluster.yaml#L85). In aws-operator it statically set in installations repo.

no more comments.